internal/keygen/keygen.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/vultisig/vultisigner/relay"
 )
 
-func JoinKeyGeneration(kg *types.KeyGeneration) (string, string, error) {
+func JoinKeyGeneration(kg *types.KeyGeneration) (string, string, []string, string, string, error) {
```

Comment:
Maybe it will be better just do the backup in `JoinKeyGeneration` function , as soon as keygen finish successfully
Thus you don't need to return a lot stuff